Makefile

```diff
@@ -81,27 +81,30 @@ hr-docs: ## Generate Docusaurus documentation and blog posts
 SUPERDUPERDB_EXTRAS ?= dev
 
 testenv_image: ## Build a sandbox image
-	@echo "===> Build superduperdb/sandbox"
-	docker build . -f deploy/images/superduperdb/Dockerfile -t superduperdb/sandbox --progress=plain \
+	@echo "===> Build superduperdb/$(SUPERDUPERDB_EXTRAS)"
```

Comment:
This allows to maintain 2 images and deploy them in 2 slightly different ways in the sandbox environment.